apps/condo/domains/billing/components/OnBoarding/index.tsx

```diff
@@ -43,21 +47,36 @@ export const BillingOnboardingPage: React.FC<BillingOnboardingPageProps> = ({ on
     const router = useRouter()
     const [welcomeShown, setWelcomeShown] = useState(false)
 
-    const [currentStep] = useOnboardingProgress()
+    useEffect(() => {
+        setWelcomeShown(Boolean(localStorage.getItem(MODAL_STORAGE_KEY)))
+    }, [])
+
+    const [currentStep] = useOnboardingProgress(withVerification)
 
     const stepItems: Array<StepItem> = useMemo(() => {
-        return [
+        const steps: Array<StepItem> = [
             { title: BillingSelectTitle },
             { title: SetupBillingTitle, breakPoint: true },
-            { title: SetupAcquiringTitle },
+            { title: SetupAcquiringTitle, breakPoint: true },
```

Comment:
Seems like `breakPoint` its an redundant property. It's last element of the array, so you have no options to go further. What is the meaning of its definition?

Reply:
There can be a verification step after, so it's not redundant.